source/server/options_impl.cc

```diff
@@ -92,6 +108,9 @@ OptionsImpl::OptionsImpl(std::vector<std::string> args,
   TCLAP::SwitchArg log_format_escaped("", "log-format-escaped",
                                       "Escape c-style escape sequences in the application logs",
                                       cmd, false);
+  TCLAP::ValueArg<bool> log_format_prefix_with_location(
+      "", "log-format-prefix-with-location",
+      "Prefix all logged messages with '[path/to/file.cc:99] '.", false, true, "bool", cmd);
```

Comment:
I think it's worth writing in the option docs that this is here for backwards compatibility and will eventually be removed, and that if it's set to true AND a format string is specified, it will replace `%v`, even if the specified format string already includes `%g` or `%#`.

Reply:
Sure, I will update docs and release notes as we've agreed on the approach now.